src/client/app/graph/arrow/arrow.component.ts

```diff
@@ -26,6 +30,13 @@ export class ArrowComponent implements OnInit {
   // the last point in anchorPoints, so this property is just for convenience.
   tipPosition: ViewportCoord = {x:0, y:0};
 
+  // Whether or not tail dragging is in progress.
+  private tailDragging: boolean = false;
+  // Whether or not tip dragging is in progress.
```

Comment:
(nit)

Add vertical spacing between each field, for visual clarity?

Reply:
Really? Seems clear to me. Plus vertical space aint cheap.